### DISCUSSION TITLE
Add the ability to change the accent color

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#34] [Ideas]
@qw123wh, @odmfl & **others who come across this conversation**:

Do you think it would be a good idea to be able to change the accent color of the interface?
I made the changes in this [branch](https://github.com/BlackyHawky/Clock/tree/Accent_color).
I await your comments if you have time to test.

Please note: I cannot change the color of the icon in notifications and the background color of the icons when long-pressing the app icon. Surely there is a way to do this but I can't find what it is.

**Comment by odmfl** [comment]
@BlackyHawky Fantastic, if you also add the transparent background color which should be #00FFFFFF you have also solved the issue #29 setting the background color transparent = bye bye material you. I'm a genius🤣🤣🤣🤣

**Reply by BlackyHawky** [reply]
This addition is therefore satisfactory. I'm happy to read it. 😁
However, if I add the "Transparent" style, we remove the ability to remove backgrounds from the various elements when choosing an accent color. That's why I'll add a specific setting. 😉

**Comment by qw123wh** [comment]
yes excellent function, you could also think about changing the app icon, the one on fdroid is the same as google clock, you could make a new icon if you want.

**Reply by BlackyHawky** [reply]
> ... you could also think about changing the app icon ...

Unfortunately, I'm not experienced enough to create icons. 
Hopefully, someone with an interest in the application will come up with a new one.
